src/WebJobs.Script.WebHost/WebScriptHostBuilderExtension.cs

```diff
@@ -52,8 +53,6 @@ public static class WebScriptHostBuilderExtension
                 })
                 .AddScriptHost(webHostOptions, configLoggerFactory, metricsLogger, webJobsBuilder =>
                 {
-                    webJobsBuilder.AddAzureStorageCoreServices();
```

Comment:
Is the scope of this registration changing? Just want to make sure we validate the other services without this registration here. Was it moved elsewhere?

Reply:
Didn't come across anything in my manual validation but yes, we are changing the scope. When there is a WebHost, it is registered in this parent - if not, registered in the ScriptHost container.